tests/internal/cluster/cluster.go

```diff
@@ -171,3 +179,209 @@ func ExecCmdWithStdout(
 
 	return outputMap, nil
 }
+
+// new commands
+
+// ExecLocalCommand runs the provided command with the provided args locally, cancelling execution if it exceeds
+// timeout.
+func ExecLocalCommand(timeout time.Duration, command string, args ...string) (string, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), timeout)
+	defer cancel()
+
+	glog.V(params.LogLevel).Infof("Locally executing command '%s' with args '%v'", command, args)
+
+	output, err := exec.CommandContext(ctx, command, args...).Output()
+
+	return string(output), err
+}
+
+// ExecCmdWithRetries executes a command on the provided client on each node matching nodeSelector,
+// retrying on internal errors
+// retries times with a 10 second delay between retries, and ignores the stdout.
+func ExecCmdWithRetries(client *clients.Settings, retries uint, nodeSelector, command string) error {
+	for retry := range retries {
+		err := ExecCmd(client, nodeSelector, command)
+		if isErrorExecuting(err) {
+			glog.V(params.LogLevel).Infof("Error during command execution, retry %d (%d max): %w", retry+1, retries, err)
+
+			time.Sleep(10 * time.Second)
```

Comment:
It'll be good to add sleep interval as a argument.

Reply:
fixed